src/components/molecules/QuizCta/QuizCta.tsx

```diff
@@ -28,7 +28,11 @@ const StyledQuizCta = styled("div")<StyledQuizCtaCardProps>`
 
   *:not(button) {
     color: ${(props) =>
-      props.lightContrast ? props.theme.white : props.theme.gray1};
+      props.theme.theme === "custom"
```

Comment:
`props.theme.theme` would appear only in stylegist context 
so we can't use it in the components 

try decrease the value 

`contrast("#fff", theme.primaryColor) >= 4.5;`

Reply:
I decreased the contrast ratio to the blue variant which had different requirements and works fine